tools/provision.sh

```diff
@@ -174,3 +168,10 @@ case "${AUTH_METHOD}" in
         sudo systemctl start osbuild-composer.socket
         ;;
 esac
+
+# Print debugging info about content sources
+sudo composer-cli status show
```

Comment:
This should be moved under the `AUTH_METHOD_NONE` section, tests meant for the service don't have Weldr enabled at all.